packages/arb-token-bridge-ui/src/components/TransferPanel/TokenSearch.tsx

```diff
@@ -429,7 +429,7 @@ function TokensPanel({
   }, [tokensFromLists, tokensFromUser, newToken, getBalance])
 
   const storeNewToken = async () => {
-    return token.add(newToken).catch((ex: Error) => {
+    return token.add(newToken.toLowerCase()).catch((ex: Error) => {
```

Comment:
Maybe it's better to do this here: https://github.com/OffchainLabs/arb-token-bridge/blob/master/packages/token-bridge-sdk/src/hooks/useArbTokenBridge.ts#L828

Otherwise we'd have to do it every time we call the function

Reply:
Agreed, done